content/99.back-matter.md

```diff
@@ -1,6 +1,7 @@
 ## Acknowledgements
 
-`TODO: deep review authors for support in testing this process`
+We would like to thank the authors of the Deep Review who helped us test this process.
```

Comment:
How about `test collaborative writing with Manubot` instead of `test this process`?